quickstarts/angular1/01-login.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Setup and Authentication
+name: Angularjs-login
+alias:
+  - angular
+  - angularjs
+language:
+  - Javascript
+framework:
+  - AngularJS
+image: /media/platforms/angular.png
+tags:
+  - quickstart
+  - angular
+snippets:
+  dependencies: /quickstarts/angular1/snippets/dependencies
+  init: /quickstarts/angular1/snippets/init
+  head: /quickstarts/angular1/snippets/head
+alias:
+  - angular
+---
+
+## Setup and Authentication
+
+One of the most aching aspects of an Angular App is authentication. It becomes worse when you have routes to protect, page refresh to deal with, tokens to manage, multi-factor authentication, and so on. This quickstart is planned to help you tackle these challenges.
```

Comment:
I think this part isnt needed. Our customers have come to our site specifically to solve the issue of authentication so I think this is a bit redundant.